app/scaffold-controllers/navigationController.js

```diff
@@ -51,11 +65,15 @@ function(
                 navigationHeaderController.registerDrawerEvents(drawerEventHandler);
             }
 
+            searchBarController.addToLayout();
+            navigationHeaderController.registerSearchBarEvents(searchBarController.toggle.bind(searchBarController, true));
```

Comment:
`searchBarController.toggle.bind(searchBarController, true)`, maybe capture this in a variable as well for readability?

```
var searchBarToggle = searchBarController.toggle.bind(searchBarController, true);
navigationHeaderController.registerSearchBarEvents(searchBarToggle);
```